engine/schema/src/main/resources/META-INF/db/schema-41310to41400.sql

```diff
@@ -53,6 +53,33 @@ ALTER TABLE `cloud`.`vm_instance` ADD COLUMN `backup_offering_id` bigint unsigne
 ALTER TABLE `cloud`.`vm_instance` ADD COLUMN `backup_external_id` varchar(255) DEFAULT NULL COMMENT 'ID of external backup job or container if any';
 ALTER TABLE `cloud`.`vm_instance` ADD COLUMN `backup_volumes` text DEFAULT NULL COMMENT 'details of backedup volumes';
 
```

Comment:
This schema change has to be moved to schema-41400to41500.sql.
@Pearl1594  please raise a separate PR as this is already merged.